help/manage-user-groups.md

```diff
@@ -2,6 +2,8 @@
 
 {!user-groups-intro.md!}
 
+{!user-groups-applications.md!}
```

Comment:
This include file doesn't exist, so that's why the test is failing. The `/help/manage-user-groups` page returns an error instead of loading.

Reply:
I expect @alya forgot to `git add` a newly created file.

Reply:
oh, heh, yep!